sim/genome.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+class SimulatedGenome:
+    """ Simulated genome data generated by simuG  """
+
+    def __init__(self, name, genome_path, snp_table_path, snp_vcf_path, indel_vcf_path):
+        # Validate
+        self.assert_path_exists(genome_path)
+        self.assert_path_exists(snp_table_path)
+        self.assert_path_exists(snp_vcf_path)
+        self.assert_path_exists(indel_vcf_path)
+
+        # Assign
+        self.genome_path = genome_path
+        self.snp_table_path = snp_table_path
+        self.snp_vcf_path = snp_vcf_path
+        self.indel_vcf_path = indel_vcf_path
+        
+        self.name = name
```

Comment:
Is there an argument for making these @property?

Reply:
How come? Thinking of generating names based on filenames?

Reply:
No, nothing like that. Just that maybe we don't want to to change this attribute after initialising the object. Similar to the name property in VcfSample and RandomSample. 

Happy with keeping it as is though!

Reply:
oh gotcha. Yeah I think it's awkward have an automated naming convention consistent with those objects with the way things are currently setup.